index.css

```diff
@@ -0,0 +1,3 @@
+.first-header {
+  color: #54a;
+}
```

Comment:
You have done great at utilizing the Microverse readme template. For more professional readme documentation, please make sure to do the following:
    - customize this template making sure to have all main headers from the [README template](https://github.com/microverseinc/readme-template) are present. The only optional section is FAQ.
    - make sure to added the section Future Features, including 1-3 potential features to be added.
    - remove the section descriptions, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.

> I have commented on this file because the change for the readme file isn't contained in this pull request.

Reply:
Hello @emmiiorji , 
A change was required on the README file. And I have worked on the given comment and edited the README file as requested.
Thank you